recipes/default.rb

```diff
@@ -200,10 +200,8 @@
   end
 end
 
-apache_service_name = node['apache']['service_name']
-
 service 'apache2' do
-  service_name apache_service_name
+  service_name service_name
```

Comment:
Can you call a service_name with a variable named service_name?  That is super meta if so.  Maybe make it "#{service_name}" to make it clearer what that one is a string variable and one is a key word?

Reply:
The helper would need to be called something else really., the interpolation would fail the rubocop